parse_test.go

```diff
@@ -100,6 +134,38 @@ func Test_parseFile(t *testing.T) {
 
 	}
 }
+func Test_parseFileWithParenComment(t *testing.T) {
```

Comment:
Why separated test case?

Reply:
because we want to set the parenComment flag as true here

![image](https://user-images.githubusercontent.com/7692245/102860575-19339d80-4469-11eb-8566-81165b819061.png)